content/App.qml

```diff
@@ -14,11 +14,82 @@ Window {
     MainScreeen {
         id: mainScreen
 
-    }
+        MenuButton {
+            id: menuButton
+        }
 
-    MenuButton {
-        id: menuButton
-    }
+        Gauges {
+            id: gauges
+
+            width: parent.width * 0.47

```

Comment:
maybe extract width and height, anchors to new file eg. Constrant.qml and define readonly properties there. imo is better  and easer to change values in one place :)
and this 0.47 value will be hidden